tsconfig.json

```diff
@@ -17,17 +17,18 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "baseUrl": ".",
     "paths": {
-      "@/*": [
-        "./src/*"
-      ]
+      "@/*": ["src/*"],
     },
     "forceConsistentCasingInFileNames": true
   },
   "include": [
     "next-env.d.ts",
     "**/*.ts",
-    "**/*.tsx"
+    "**/*.tsx", 
+    "jest.config.mjs", 
+    "__tests__/index.test.js"  
```

Comment:
Update this to `__tests__/**/*.test.ts` and add a similar one for `.tsx`. This way, it captures any subdirectories and any files that end in `.test.ts` or `.test.tsx`.